handlers/main.yml

```diff
@@ -2,7 +2,6 @@
 # handlers file for tinc
 
 - name: Restart Service
-  systemd:
-    daemon_reload: yes
-    state: restarted
+  service:
     name: "{{ service_name }}"
+    state: restarted
```

Comment:
AFAIK, daemon_reload still need to be provided for systemd purposes (and omitted in non systemd cases)

Reply:
The service action plugin passes the daemon_reload option automatically: https://github.com/ansible/ansible/blob/v2.7.5/lib/ansible/plugins/action/service.py#L64 

This should work:
`
service:
  name: "{{ service_name }}"
  daemon_reload: "{{ ansible_service_mgr == 'systemd' | ternary('yes',omit) }}"
  state: restarted
`

Reply:
That's a neat feature.   Would be good to have in the docs! :)  I updated to this.

Reply:
That's weird, which version of Ansible do you have, and are you sure omit is not between quotes?